test/packages/good/manifest.yml

```
@@ -40,3 +40,6 @@ icons:
     title: system
     size: 1000x1000
     type: image/svg+xml
+buildDependencies:
+  - name: ecs
+    reference: git@7a11f3e1f7d8f05e8f06623fd98254b8cb34c441
```

I wonder if there should be a `build` object, in case there are more build options in the future.
```suggestion
build:
  dependencies:
  - name: ecs
    reference: git@7a11f3e1f7d8f05e8f06623fd98254b8cb34c441
```

---

Hm.. sounds like a more future-oriented design. Let me adjust the schema.

---

Fixed